setup.sh

```diff
@@ -442,6 +437,14 @@ fi
 echo "-----------------------------------------"
 echo ""
 
+# Create Grafana Settings if missing (required in 2.4.0)
+if [ ! -f ${GF_ENV_FILE} ]; then
+    cp "${GF_ENV_FILE}.sample" "${GF_ENV_FILE}"
+
+    echo ""
+    ./import-dashboards.sh
+fi
+
```

Comment:
Similar to the comment on the other PR, I think we need to assume that people will run `setup.sh` to change their configuration so `import-dashboard.sh` should be able to run against existing installations (or even standalone). For this case, I think you could probably just have it check for imported dashboards and prompt the user to see if they want to overwrite the existing versions.